src/test/java/net/pms/test/formats/FormatTest.java

```diff
@@ -21,6 +21,17 @@
 
 import ch.qos.logback.classic.LoggerContext;
 import net.pms.formats.*;
+import net.pms.formats.audio.AIFF;
+import net.pms.formats.audio.FLAC;
```

Comment:
@Sami32 I haven't done the tests for this PR yet, I do that as the last thing so I don't have to change them many times as things change.